deepforest/utilities.py

```diff
@@ -245,24 +245,33 @@ def xml_to_annotations(xml_path):
 def shapefile_to_annotations(shapefile,
                              rgb,
                              buffer_size=0.5,
-                             convert_to_boxes=False,
+                             geometry_type="boundingbox",
                              savedir="."):
     """
     Convert a shapefile of annotations into annotations csv file for DeepForest training and evaluation
+
+    Geometry Handling:
+    The geometry_type is the form of the objects in the given shapefile. It can be "boundingbox" or "point".
+    If geometry_type is set to "boundingbox" (default) then the bounding boxes in the shapefile will be used as is and transferred over
+    to the annotations file. However, if the geometry_type is "point" then a bounding box will be created in its place that's centered on
+    the original point with an apothem equal to buffer_size.
```

Comment:
Just for a little added specificity how about changing the last sentence to something like:

> If the geometry_type is "point" then a bounding box will be created for each point that is centered on the point location and has an apothem equal to buffer_size, resulting in a bounding box with dimensions of 2 times the value of buffer_size.